.ci-operator.yaml

```diff
@@ -0,0 +1,4 @@
+build_root_image:
+  name: release
+  namespace: openshift
+  tag: rhel-8-release-golang-1.17-openshift-4.11
```

Comment:
how about using 1.18 here because the code needs 1.18 right?

Reply:
go.mod file can have a lower version anyways but we can stick to golang 1.17 as of now.